sql/build.gradle

```diff
@@ -1,5 +1,6 @@
 apply plugin: 'idea'
 apply plugin: 'java'
+apply plugin: 'de.thetaphi.forbiddenapis'
```

Comment:
would be nice to enable it on every java sub-project, specially if you already started to add missing locales on them (afaik you did at blob, core, dns-discovery)